app/components/survey-link/survey-link.tsx

```diff
@@ -4,7 +4,7 @@ import { ContentLink } from "../content-link"
 
 export function SurveyLink() {
   const onPressLink = () => {
-    // Linking.openURL() TODO: add real URL
+    Linking.openURL("https://www.surveymonkey.com/r/ChainReact2019 ")
```

Comment:
Is this space intentional?

```suggestion
    Linking.openURL("https://www.surveymonkey.com/r/ChainReact2019")
```

Reply:
Nope! Good catch!